### ISSUE TITLE
A Tradenode re-work

### ISSUE THREAD

**Issue by LuxyLucy** [Victorum-Universalis-VU/Victorum-Universalis#9]
Hello, so trade is one of the mechanics in the game I really enjoy to work with. For me its a big reason I go to war in multiplayer. To build a strong economy and destroy the economies of others its in my opinion the most important game mechanic that drives the game forward.
I want to suggest a tweak to the trade nodes that would really bring alive the game more in my eyes.
I want to first of all make the trade node of Beijing an end node. Throughout History the true and final destination of all trade was with the Chinese, what we found to be important items was due to the Chinese wanting of them. Silver for example was one of the few items the Chinese would trade for, along with copper.
By doing that Beijing would be turned into a powerful trade node that trade through Asia could be pulled into, but say if a European power took over through Siberia like for example Russia, or the Mughals came through Tibet. The same nodes that feed into Beijing also feed into Xi'an so those nations could still pull trade from China to them through Xi'an so it doesn't remove that option from them while helping represent the importance of the Chinese market. Though Yumen as well should be made to be able to flow into Beijing.
Next, the English Channel trade node should be split. England and the Netherlands receiving their own separate end nodes to create a truly interesting conflict between them. Instead of them both just trying to fit as many trade ships as possible into the English Channel to extract more money than the other, Have them compete over the flow into their nodes in a truly grand scale world wide battle for trade dominance where they compete in Africa and the Americas as well as Europe for more power in diverting trade into their end nodes. Instead of the both of them flowing trade into the English Channel.
For this, The North Sea instead of flowing into Cologne would flow into the Netherlands, England and Lubeck. Lubeck should also flow into the England node while also flowing into the Cologne node, the Cologne node would flow into Champagne but also the Netherlands node. Champagne would flow into the Netherlands and Rouen would flow into the England node and the Netherlands node. And finally the Carribean node should flow into the England node and the Netherlands node.
Thank you for reading my suggestion, I really believe this would create interesting conflict throughout the world for power between these two in pulling trade into their respective nodes, and giving China more historical authenticity.

**Comment by slamchan** [comment]
I agree with Beijing and having own nodes for Enland and The Netherlands. Trade flows need to be thought about. Very good enhancement idea!

**Comment by slamchan** [comment]
I have made Beijing an end node and made Yumen flow into it. There is a graphical bug regarding this route, but that is not first priority (for me) since it just wörks.

**Comment by PirateAE** [comment]
While this would be a good idea, in practice its not viable, if trade routes could change directions/being end node or not mid game, the i would be all for this but until that becomes a feature, im gonna have to hard disagree to this idea.
